cypress/integration/PressReleases/index.js

```diff
@@ -0,0 +1,63 @@
+/// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+And('the 2019 release doesnt have a link', () => {
+    cy.get('p').should('contain.text', '2019').should('not.have','a');
+});
+
+And('the following press releases are displayed', (dataTable) => {
+    for (const { title, link } of dataTable.hashes()) {
+        cy.get('p').find(`a:contains('${title}')`).should('have.attr', 'href', link);
+    }
+});
+
+And('{string} box is displayed with a link {string}', (linkText, linkHref) => {
+    cy.get('.callout-box-full').find('h3').should('have.text', linkText);
+    cy.get('.callout-box-full').find('a').should('have.attr', 'href', linkHref);
+    
+});
+
+And('the list of press release pages is displayed', () => {
+    cy.get('.views-element-container').should('exist');
+});
+
+And('each title is a link with href that starts with {string}', (itemHref) => {
+    cy.get('.views-element-container > ul> li').find('a').should('have.attr', 'href').then((link) => {
+        expect(link).to.include(itemHref)
+    })
+});
+
+And('each date is for year {string}', (year) => {
+    cy.get('.views-element-container > ul> li').find('time').should('contain.text', year);
+});
+
+And('posted date is not displayed', () => {
+    cy.get('.clearfix').should('not.exist');
+});
+
+And('the contact {string} is displayed with a link to email {string}', (pressText, email) => {
+    cy.get('.columns.large-4.media-contact.right a').should('have.attr', 'href', email).should('have.text', pressText);
+});
+
+And('user is clicking on definition for {string}', (glossTerm) => {
+    cy.get('.definition').click();
```

Comment:
What is this step? It is not in a feature file

Reply:
updated this.

Reply:
The step is still here - delete it please